src/components/Timeline/Event.js

```diff
@@ -52,15 +52,13 @@ export const Event = ({
                 <CardHeader
                   title={ticketId ? `${ticketId}: ${text}` : text}
                   subtitle={timeFormattedToMultipleZones(time)}
-                  actAsExpander
-                  showExpandableButton
                   iconStyle={{
                     color: isAllPlaybookInfoAvailable ? 'black' : 'Lightgrey'
                   }}
             />
                 {
               isAllPlaybookInfoAvailable &&
-              <CardText expandable>
```

Comment:
This is a functional change, let's not mix it in with removing unused code.

Reply:
The main goal of this PR was to remove the expand/collapse functionality. The ticket compare removal was just piggy backed here. I can split the PR in two if needed.